src/crate/crash/command.py

```diff
@@ -402,6 +441,8 @@ def _connect(self, server):
             self.logger.critical('CONNECT ERROR')
         else:
             self.logger.info('CONNECT OK')
+        if connected:
```

Comment:
the `connected` is not necessary, because you an put the `check()` call into the `else` block